### ISSUE TITLE
ASP.NET Identity needs it's own database

### ISSUE THREAD

**Issue by frankhale** [DIOS-Tech/BlueHarvest#57]
The first time a user is registered through ASP.NET Identity a series of database tables are built to store the information. Right now if one were to register a new user those tables would go into our database. These tables are rather ugly, not meant to be touched and will just pollute our database and make it harder to manage. 

What I think should happen is that ASP.NET Identity needs to have it's own database. This is simple to do. Just create another connection string in the web.config and then change the following in Infrastructure.fs to know about the new connection string:

```fsharp
type ApplicationDbContext() =
  inherit IdentityDbContext<ApplicationUser>("DefaultConnection", throwIfV1Schema = false)
```

**Comment by frankhale** [comment]
Boo, user registration is broken. This is a regression, it used to work. DAMN!

**Comment by frankhale** [comment]
Fixed via ec414a65be3771db912bb1cf7f17653f2fa9bc2b

**Comment by ybotspawn** [comment]
Nice!
